### DISCUSSION TITLE
继续部署后报bug 😂

### DISCUSSION THREAD

**Discussion by aozaki-kuro** [spencerwooo/onedrive-vercel-index#54] [General]
大佬您好，我又来了，继续报bug

这次问题主要集中在`README.md`中

## font awesome无法使用

目前尽管可以解析HTML代码，但是font awesome仍然没有办法正确引入。

期望：能够在`README.md`中使用font awesome。
测试页面：https://beta.suisei.cc/%E2%9C%A83D%20Live

![image](https://user-images.githubusercontent.com/28428951/129524326-b9da3b6a-b962-4259-aa8d-a5e2fd93c070.png)

## img标签占位不正常

`README.md`中插入图片后，<img>会占满一整栏，同一行插入的第二张图片会被挤到第二行去。
连带使用的超链接也会在图片之外的区域被触发。

期望：<img>元素不会占满一整行，超链接只在图片区域被触发。可以并排插入第二张图片。
测试页面：https://beta.suisei.cc/%F0%9F%8E%ACMV

![image](https://user-images.githubusercontent.com/28428951/129524366-1b74dde2-8e8c-4d49-aebc-5b9e189edda7.png)

## 意外重载

`README.md`在加载完成后，可能在切换文件夹时触发重载。首次进入该文件夹时不会触发。

复现方法：
1. 进入文件夹A，等待加载完成。
2. 从文件夹B后再回到文件夹A，触发`README.md`重载

这个bug尽管不影响功能，但是那么大面积内容进行闪烁和重载在观感上非常难受。
上一个版本似乎是没有的，希望能看一下问题出在哪里😂

![3](https://user-images.githubusercontent.com/28428951/129527481-a58a011d-5f2b-4bf1-be43-020134e6e324.gif)

**Comment by spencerwooo** [comment]
1. Font Awesome 在本项目里是使用 [Font Awesome React component](https://github.com/FortAwesome/react-fontawesome)，可以只引入需要的图标，不需加载整个依赖。你想要在 Markdown 里面使用 Font Awesome 和本项目功能性无关了，标准 Markdown 同样也不支持 Font Awesome 所以本项目不会考虑这个功能。我建议你考虑在需要引入图标的 Markdown 文件里加上 Font Awesome CDN 链接，或许可以满足你的使用需求。
2. 渲染 img 标签是 [React Markdown](https://github.com/remarkjs/react-markdown)，我更新后能够在 Markdown 内显示 raw HTML 也仅仅是关闭了 React Markdown 的 HTML 过滤器，因此你这里的特殊功能同样也不是本项目准备考虑的。
3. 这个是 useSWR 的特性，也就是在一个页面重新进入视野后自动刷新 API content，这个倒是可以关掉，但是 Next.js 官方不建议。

**Reply by spencerwooo** [reply]
换到 discussion 里面讨论吧，我觉得除了 3，别的至少我不知道怎样在不影响本项目整体 integrity 的前提下做调整。

**Reply by spencerwooo** [reply]
个人觉得你这个需求特殊，可以考虑自己制作个网站实现

**Reply by aozaki-kuro** [reply]
谢谢解答🤣自建网页实在没这个能力，所以还是先修修改改用着了。
1和2确实不算特别大的问题，稍加修改我已经做好替代了。
就是3毕竟影响体验，想知道一下我fork出去后怎么关？

**Comment by spencerwooo** [comment]
最新版本应该没有 3 这个功能了，关掉了 `useSWR` 里面一些非必要的 revalidation 功能。

**Comment by tkfanswind** [comment]
README.md里的图片缩放好像不好使。总是显示原图，特别大。如以下代码，不缩放到60%`

` <img` src="https://img2.doubanio.com/view/photo/l/public/p2678888032.jpg" width="60%" />
